crates/shim-sgx/src/heap.rs

```diff
@@ -107,7 +108,10 @@ mod tests {
 
             for allocated in steps {
                 let mut heap = Heap::new(Address::new(0), Address::new(BYTES));
-                assert_ne!(heap.mmap(Offset::from_items(allocated), Access::READ), None);
+                assert_ne!(
+                    heap.mmap(None, Offset::from_items(allocated), Access::READ),
+                    None
+                );
```

Comment:
Avoid the wrapping here by breaking this into two lines with an intermediate variable.

You should always consider an extra variable name if it avoids wrapping.

Reply:
fixed